.github/actions/setup/action.yml

```diff
@@ -8,14 +10,14 @@ runs:
       with:
         node-version: 14
         registry-url: https://registry.npmjs.org
-        cache: yarn
```

Comment:
why are you removing this? it is probably an noop for us but it says it caches global packages data, not node_modules https://github.com/actions/setup-node#caching-global-packages-data

Reply:
couldn't find those docs, thought it was duplicating the node_modules cache.